main.css

```diff
@@ -0,0 +1,789 @@
+    left: 40px;
+}
+
+.header__glass-source {
+    width: 35px;
+    height: 35px;
+    position: absolute;
+}
+
+.header__small-container {
+    display: flex;
+    height: 220px;
+    justify-content: center;
+    align-items: center;
+    gap: 40px;
+}
+
+.header__image {
+    height: 180px;
+    width: 250px;
+    display: flex;
+}
+
+.header__burger {
+    height: 60px;
+    width: 60px;
+    display: flex;
+}
+
+.header__textarea {
+    width: 530px;
+    height: 100px;
+    border-radius: 35px;
+    border: 1px solid black;
+    font-size: 45px;
+    text-align: center;
+    padding-top: 50px
+}
+
+/*let's ride section  */
+
+.let-ride {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    gap: 80px;
+    padding: 100px 0px 100px 0px;
+    font-size: 100px;
+    font-weight: 550;
+}
+
+.let-ride__header {
+    font-size: 100px;
+    font-weight: 550;
+}
+
+.let-ride__buttons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.let-ride__apply-button {
+    line-height: 2.5;
+    padding: 0px 70px 0px 70px;
+    font-size: 40px;
+    text-align: center;
+    color: #FFFFFF;
+    border-radius: 200px;
+    background-color: #523be4;
+    font-weight: 600;
+}
+
+.let-ride__sign-button {
+    line-height: 2.5;
+    padding: 0px 70px 0px 70px;
+    font-size: 40px;
+    text-align: center;
+    color: #000;
+    border-radius: 200px;
+    background-color: #FFFFFF;
+    border: 3px solid grey;
+    position: relative;
+    font-weight: 600;
+}
+
+.let-ride__image{
+    width: 900px;
+    height: 500px;
+}
+
+/* Drive - with section */ 
+
+.drive-with {
+    display: flex;
+    text-align: center;
+    flex-direction: column;
+    gap: 50px;
+    background-color: #F4F4F9;
+    padding: 200px 0px 200px 0px;
+}
+
+.drive-with-lyft__header {
+    font-size: 40px;
+    font-weight: 600;
+    width: 100%;
+}
+
+.drive-with-set__header {
+    font-size: 70px;
+    font-weight: 600;
+    width: 100%;
+    line-height: 1.2;
+}
+
+.drive-with__list {
+    display: flex;
+    flex-direction: column;
+    gap: 80px;
+    text-align: left;
+    justify-content: left;
+}
+
+.drive-with__small-container {
+    padding-left: 40px;
+    gap: 30px;
+    display: flex;
+    flex-direction: row;
+    line-height: 1.5;
+}
+
+.drive-with__text-section {
+    display: flex;
+    flex-direction: column;
+}
+
+.drive-with__text {
+    font-size: 40px;
+    font-weight: 300;
+}
+
+.drive-with__bold {
+    font-size: 40px;
+    font-weight: 600;
+}
+
+.drive-with__icon {
+    width: 50px;
+    height: 50px;
+}
+
+.drive-with__buttons {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 100px;
+}
+
+.drive-with__how-driver-pay-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.drive-with__bold {
```

Comment:
mixed up element and modificator, read https://en.bem.info/methodology/naming-convention/